src/views/Login.vue

```diff
@@ -236,6 +236,7 @@ export default defineComponent({
         return
       }
 
+      this.presentLoader('Logging in')
```

Comment:
We need to handle the case where user cannot dismiss the loader on backdrop click

Reply:
@amansinghbais

Reply:
The presentLoader method used here, by default not allow user to dismiss the loader on backdrop click.